pkg/aws/gw-machineset.yaml

```diff
@@ -4,7 +4,7 @@ kind: MachineSet
 metadata:
   labels:
     machine.openshift.io/cluster-api-cluster: {{.InfraID}}
-  name: {{.InfraID}}-submariner-gw-{{.AZ}}
```

Comment:
This commented out on purpose?

Reply:
crap - I didn't mean that - thanks for catching it.  The yaml linter was failing on that line due to the "{{*}}" part so I was playing around with it.  I think it's a false positive and I ended up taking out that linter.